our-app/src/components/unauthenticated/SignupForm.js

```diff
@@ -19,7 +19,8 @@ const SignupForm = (props) => {
         }
 
         post('/user', user)
```

Comment:
I will move this into actions after this is merged.

Reply:
Sounds good